starboard/android/arm/args.gn

```diff
@@ -15,3 +15,4 @@
 target_platform = "android-arm"
 target_os = "android"
 target_cpu = "arm"
+v8_enable_webassembly = true
```

Comment:
The flag should be turned on in starboard/linux/shared/platform_configuration/configuration.gni and andstarboard/android/shared/platform_configuration/configuration.gni

The args.gn is only used for local builds.